packages/widgets/src/media-server/component.tsx

```diff
@@ -150,6 +151,16 @@ export default function MediaServerWidget({ integrationIds, isEditMode }: Widget
       onClick: () => {
         openModal({
           item: row.original,
+          title:
+            row.original.currentlyPlaying?.type === "movie" ? (
+              <IconMovie size={36} />
+            ) : row.original.currentlyPlaying?.type === "tv" ? (
+              <IconDeviceTv size={36} />
+            ) : row.original.currentlyPlaying?.type === "video" ? (
+              <IconVideo size={36} />
+            ) : (
+              <IconDeviceAudioTape size={36} />
+            ),
```

Comment:
```suggestion
        },
        {
          title:
            row.original.currentlyPlaying?.type === "movie" ? (
              <IconMovie size={36} />
            ) : row.original.currentlyPlaying?.type === "tv" ? (
              <IconDeviceTv size={36} />
            ) : row.original.currentlyPlaying?.type === "video" ? (
              <IconVideo size={36} />
            ) : (
              <IconDeviceAudioTape size={36} />
            ),
        },
```

I would use the official modal title instead of passing it as innerProps

Reply:
i get an error - Type 'Element' is not assignable to type 'stringOrTranslation | undefined'.

Reply:
Okay I'll fix that real quick 👍🏼

Reply:
Okay I don't do it, I like the look of your implementation more